src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/Example1.java

```diff
@@ -0,0 +1,41 @@
+/*
+WhitespaceAfter
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.whitespaceafter;
+
+import java.io.InputStream;
+
+class Example1 {
+  // xdoc section -- start
+  void example() throws Exception {
+    if (true) {
+    } else if(false) { // violation 'not followed by whitespace'
+    }
+    testOne("x", "y");
+    testOne("z","o"); // violation 'not followed by whitespace'
+    for (int i = 0; i < 10; i++){}
+    for(int i = 0; i < 10; i++){} // violation 'not followed by whitespace'
+    try (InputStream ignored = System.in) {}
+    try(InputStream ignored = System.in) {} // violation 'not followed by whitespace'
+    try {} catch (Exception e){}
+    try{} catch (Exception e){} // violation 'not followed by whitespace'
+    try {} finally {}
+    try {} finally{} // violation 'not followed by whitespace'
+    try {} catch (Error e){} finally {}
+    try {} catch (Error e){} finally{} // violation 'not followed by whitespace'
+    try {} catch (Exception e){}
+    try {} catch(Exception e){} // violation 'not followed by whitespace'
+    synchronized (this) { }
+    synchronized(this) { } // violation 'not followed by whitespace'
+  }
+  public String testOne(String a, String b) {
+    return (a + b);
+  }
+  public String testTwo() {
+    return("a" + "b"); // violation 'not followed by whitespace'
+  }
+  // xdoc section -- end
```

Comment:
Why did we remove switch expression with example of yield statement?

Reply:
Let's increase 45 line limit to what it is required by a example.
45 is not a limit we did on good reason, we can estimate better good size at the end.

Reply:
PR is assigned to me, hmm..

Do we have support of not compile able in examples ? If yet please return switch and extend limit  to avoid violation.

Reply:
Done. Moved to resources-noncompilable.

Reply:
Forgot compilation comment.

Reply:
Added.

Reply:
@nrmancuso , please finish review